tests/runtest.cmd

```diff
@@ -80,6 +81,7 @@ echo     /determinism  : Compile the test twice with randomized dependency node
 echo                      compiler determinism in multi-threaded compilation.
 echo     /nocleanup    : Do not delete compiled test artifacts after running each test
 echo     /r2rframework : Create ready-to-run images for the CoreCLR framework assemblies
+echo     /gcstresslevel: GC stress level to use for testing
```

Comment:
This flag only available to CoreCLR-targeting CPAOT. I think we should make that clear in the usage like you did with `/r2rframework`. Or perhaps group these CPAOT-specific flags under a "CoreCLR ReadyToRun" header?

Reply:
Good suggestion. For now I have put the option (together with r2rframework) under a "CPAOT-specific flags" section. We can further iterate on this based on whatever shipping name for CPAOT we end up with.